### DISCUSSION TITLE
Phonetype of the progenies

### DISCUSSION THREAD

**Discussion by Dkplucas** [gaynorr/AlphaSimR#100] [Q&A]
Recently, during a presentation about AlphasimR, my professor ask me what is the formula used by AlphasimR to obtain the phenotypes of the progenies from the phenotypes and the genetic values of the sire and dam. I failed to answer cause I only know the formula used by AlphasimR for the population size not between a sire and a dam. Dear all and professors can you help me please to understand better?

**Comment by gregorgorjanc** [comment]
Maybe we can help with a simulation!?

Why don't you share a simple simulation here so we clear it out?

Also, this and other topics are covered extensively in a free online course at https://www.edx.org/course/breeding-programme-modelling-with-alphasimr

**Reply by Dkplucas** [reply]
Exactly professor, it is about the online course. In this code they used the breeder;s equation to determine the phenotypes expected in the progenies and the reponse to selection. So the question was to know during the mating of a sire and a dam, how does AlphasimR calculate the phenotype of the possible progenies and what is the formula used by the package?

**Reply by gregorgorjanc** [reply]
@Dkplucas I think you are missing some steps in your thought process or confusing multiple steps. AlphaSimR does not (at least not generally) simulate phenotype of an individual, based on the phenotypes of parents. What AlphaSimR does, is it simulates genomes of individuals (and genomes of a progeny come from haplotypes of its parents via recombination and segregation processes) (Week 3: DNA lottery). Once we have the genomes of progeny, it then adds up the effects of QTL to get the genetic value of an individual and adds environmental deviation to get phenotype of an individual, which in our case here are progeny of some parents (Week 2: Simulation of DNA and traits). Note that parental phenotypes are not directly involved in this simulation.

**Reply by Dkplucas** [reply]
Thanks so much professor, now I understand. I thought the parent where involved in the simulation process due to the usage of ''randcross''.

**Reply by gregorgorjanc** [reply]
@Dkplucas indeed, parental genomes generate progeny genomes. Once we have progeny genomes, we can generate their phenotypes. It's DNA (genomes) that is passed between generations.

**Reply by Dkplucas** [reply]
Ok Thank you so much professor, it is very clear now. Hope I will have the chance to meet you one day so many things to ask you. 
Please let me know if you have new classes on AlphasimR packages, I will be very happy to take part. Since I have made it my research tool.

**Comment by Dkplucas** [comment]


**Reply by gregorgorjanc** [reply]
@Dkplucas I would appreciate if you remove this specific post where you have verbatim copied the whole vignette from Week 4 of the said course. The course is for free to anyone so no need to make copies of its material around the web. Thanks!

**Reply by Dkplucas** [reply]
Sure professor, sorry for that I wanted to point it out
